example/readme.md

```diff
@@ -0,0 +1,31 @@
+# Example
+
```

Comment:
Can you add a sentence here saying that this example is meant to illustrate how a simple agent that just parrots back whatever the user is saying can be connected to ChatWidget. Also make it clear that the ParrotAgent is implemented in the DialogueKit toolkit (+link).

Reply:
Done